src/material/radio/radio.html

```diff
@@ -28,6 +18,16 @@
         [attr.aria-describedby]="ariaDescribedby"
         (change)="_onInputChange($event)"
         (click)="_onInputClick($event)">
+
+    <div mat-ripple class="mat-radio-ripple mat-focus-indicator"
```

Comment:
Can you add a comment here that explicitly calls out that the ripple comes after the input so it can be targeted with a CSS sibling selector?